pandas/_typing.py

```diff
@@ -93,20 +92,10 @@
     from typing import TypeGuard  # pyright: ignore[reportUnusedImport]
 
     P = ParamSpec("P")
-
-    if sys.version_info >= (3, 11):
-        from typing import Self  # pyright: ignore[reportUnusedImport]
-        from typing import Unpack  # pyright: ignore[reportUnusedImport]
-    else:
-        from typing_extensions import Self  # pyright: ignore[reportUnusedImport]
-        from typing_extensions import Unpack  # pyright: ignore[reportUnusedImport]
-
 else:
     ParamSpec: Any = None
-    Self: Any = None
     TypeGuard: Any = None
     Concatenate: Any = None
-    Unpack: Any = None
```

Comment:
Should test always importing `Self` and `Unpack` from `typing` whether or not we are doing `TYPE_CHECKING` and then other modules that import those symbols from `_typing` will have access to them, as they should work at runtime.

I don't think it's a good idea to change `Self` and `Unpack` to `Any` at runtime.

Reply:
OK, that works.